### PR TITLE
Polar Bear Demo part 3: readme fixes

### PR DESCRIPTION
**Pull request by blueapple128**
Performed this lab and was not able to proceed without making the following manual changes. See PR comments for details.

### CODE REVIEW

File: Labs/Deep Learning/400 - Stream Analytics and Machine Learning/3 - Predict/readme.md [hunk 1]
```diff
@@ -228,7 +228,7 @@ In this exercise, you will modify the Azure Function that you created in the pre
 1. Open the Azure Function contained in the Function App. Replace the function code with the following code:
 
 	```javascript
-	module.exports = async function (context, req) {
+	module.exports = function (context, req) {
```
Comment (blueapple128) [inline Labs/Deep Learning/400 - Stream Analytics and Machine Learning/3 - Predict/readme.md]: An async Azure Function seems to immediately exit upon returning, even if there are still asynchronous callbacks in progress. Minimal reproducible example:

```
module.exports = async function (context, req) {
    setTimeout(() => { context.log('two'); }, 1000);
    context.log('one');
};
```
will output 'one' and never output 'two'. In this case, the function never makes it past sending the request to the Custom Vision Service.

Comment (jeffprosise) [reply Labs/Deep Learning/400 - Stream Analytics and Machine Learning/3 - Predict/readme.md]: Just ran into this this week and am fixing.

File: Labs/Deep Learning/400 - Stream Analytics and Machine Learning/3 - Predict/readme.md [hunk 2]
```diff
@@ -293,8 +293,15 @@ In this exercise, you will modify the Azure Function that you created in the pre
...
-	                password: databasePassword,
+	                authentication:
+	                {
+	                    options:
+	                    {
+	                        userName: databaseUsername,
+	                        password: databasePassword
+	                    },
+	                    type: 'default'
+	                },
```
Comment (blueapple128) [inline Labs/Deep Learning/400 - Stream Analytics and Machine Learning/3 - Predict/readme.md]: This caused a lot of frustration as the original code persistently outputted `ConnectionError: Login failed for user ''.` indicating a blank username. Found the correct syntax [here](https://docs.microsoft.com/en-us/azure/sql-database/sql-database-connect-query-nodejs) under the top hit for "tedious connect to azure sql database". I didn't spend too much time digging for the origin/root cause but my first thought is a tedious package version difference?

File: Labs/Deep Learning/400 - Stream Analytics and Machine Learning/3 - Predict/readme.md [hunk 3]
```diff
@@ -321,10 +328,12 @@ In this exercise, you will modify the Azure Function that you created in the pre
 	
 	                    dbRequest.on('error', err => {
 	                        context.log(err);
+	                        context.done();
```
Comment (blueapple128) [inline Labs/Deep Learning/400 - Stream Analytics and Machine Learning/3 - Predict/readme.md]: now that the function isn't async

File: Labs/Deep Learning/400 - Stream Analytics and Machine Learning/3 - Predict/readme.md (hunk 1 above)
Comment (blueapple128) [reply Labs/Deep Learning/400 - Stream Analytics and Machine Learning/3 - Predict/readme.md]: Thanks!

File: Labs/Deep Learning/400 - Stream Analytics and Machine Learning/3 - Predict/readme.md (hunk 2 above)
Comment (jeffprosise) [reply Labs/Deep Learning/400 - Stream Analytics and Machine Learning/3 - Predict/readme.md]: Yes, I just discovered this one this week as well. It is due to a recent change in tedious as you surmised. I am in the process of updating the lab. There is also a missing step. To connect to the database from Power BI Desktop, you will need to permit your client IP through the firewall on the database server. I will be adding that step.

Comment (blueapple128) [reply Labs/Deep Learning/400 - Stream Analytics and Machine Learning/3 - Predict/readme.md]: Thanks also!

**Comment by shanamatthews** [comment]
Thanks for taking a look at this @jeffprosise - I'll leave it to you 😄

**Comment by jeffprosise** [comment]
Fix coming soon along with other fixes!

**Comment by jeffprosise** [comment]
Fixed!

**Pull request closed without merging**